core/consensus/babe/src/lib.rs

```diff
@@ -929,29 +930,13 @@ impl MaybeSpanEpoch {
 	}
 }
 
-/// Extract current epoch data from cache and fallback to querying the runtime
-/// if the cache isn't populated.
-fn epoch<B, C>(client: &C, at: &BlockId<B>) -> Result<MaybeSpanEpoch, ConsensusError> where
+/// Extract current epoch data from the runtime.
```

Comment:
There should be a TODO here for light clients.

Reply:
light client support doesn't work now. not changing the status quo